File: week 1/Mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int pyramidHeight;

    do
    {
        pyramidHeight = get_int("How height is the Pyramid?\n");
    }
    while (pyramidHeight < 1 || pyramidHeight > 8);

    for (int pyramidWidth = 1; pyramidWidth <= pyramidHeight; pyramidWidth++)
    {
        for (int hash = 1; hash <= pyramidHeight; hash++)
        {
            if (hash <= pyramidHeight - pyramidWidth)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: week 6/Mario-less/mario.py
from cs50 import get_int


while True:
    pyramidHeight = get_int("How height is the Pyramid?")
    if pyramidHeight > 0 and pyramidHeight < 9:
        break
for row in range(pyramidHeight):
    for space in range(pyramidHeight - row - 1, 0, -1):
        print(" ", end="")
    for hash in range(row + 1):
        print("#", end="")
    print("\n", end="")
